# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

HomeActionGoal action_goal
HomeActionResult action_result
HomeActionFeedback action_feedback
